src/hotspot/cpu/s390/frame_s390.hpp

```diff
@@ -96,15 +96,15 @@
 
   // ABI_160:
 
-  // REMARK: This structure should reflect the "minimal" ABI frame
-  // layout, but it doesn't. There is an extra field at the end of the
+  // REMARK: z_abi_160_base structure reflect the "minimal" ABI frame
+  // layout. There is a field in the z_abi_160
   // structure that marks the area where arguments are passed, when
   // the argument registers "overflow". Thus, sizeof(z_abi_160)
   // doesn't yield the expected (and desired) result. Therefore, as
   // long as we do not provide extra infrastructure, one should use
   // either z_abi_160_size, or _z_abi(remaining_cargs) instead of
   // sizeof(...).
```

Comment:
I would better like a wording like
Therefore, please use `sizeof(z_abi_160_base)` or the `enum` value `z_abi_160_size` to find out the size of the `ABI` structure.